### ISSUE TITLE
[SVG] SVG updates

### ISSUE THREAD

**Issue by Rumyra** [mdn/content#11597]
## Acceptance Criteria

- [x] The listed features are documented sufficiently on MDN
- [x] BCD is updated
- [x] Interactive example and data repos are updated if appropriate
- [x] The content has been reviewed as needed

### For folks helping with Firefox related documentation

- [x] Set bugs to `dev-doc-complete`
- [x] Add entry to Firefox release notes if feature is enabled in release
or
- [ ] Add entry to Firefox experimental features page if feature is not yet enabled in release

## Features to document

Couple of bugs - one is unshipping https://bugzilla.mozilla.org/show_bug.cgi?id=1388931 the other is `d` property https://bugzilla.mozilla.org/show_bug.cgi?id=1744599

I know some work has started in removal of SVG docs & `d` prop has landed on my desk before - so this should just be a case of reading through the issue & making sure docs are removed/up to date + BCD updates

NB `d` attr is currently in experimental features

**Comment by hamishwillee** [comment]
For [bug 1388931](https://bugzilla.mozilla.org/show_bug.cgi?id=1388931) I **think** it is just the following are now hidden behind `dom.svg.pathSeg.enabled`:
- `SVGPathSegList` (including .numberOfItems, getItem() and length).
  - Not documented, but is in sidebar for [SVGPathElement](https://developer.mozilla.org/en-US/docs/Web/API/SVGPathElement) and also as deprecated API in Web/API/Document_Object_Model
  - This is not in BCD.
  - Do not propose to add to either as it is on the way out. But will mention in release note.
- [SVGPathElement.getPathSegAtLength()](https://developer.mozilla.org/en-US/docs/Web/API/SVGPathElement)
  - Note in [SVGPathElement](https://developer.mozilla.org/en-US/docs/Web/API/SVGPathElement) that this was removed in SVG spec. No dedicated documented.
  - Is in BCD. Need removal
- `SVGAnimatedPathData` `.pathSegList` and `.animatedPathSegList` 
  - not documented in MDN. 
  - not in BCD

IN summary, I think all this needs is BCD note for `SVGPathElement.getPathSegAtLength()`, release note and experimental features update. No point adding BCD for the others - if they didn't need it before, they don't need it at point of removal.

Am confirming in https://bugzilla.mozilla.org/show_bug.cgi?id=1388931#c59 . For now will proceed with assumption I am correct.

Status
- [ ] BCD
- [ ] Docs
- [ ] Experimental features
- [ ] Release notes
- [ ] dev-doc-complete

**Comment by hamishwillee** [comment]
For [bug 1744599: Ship d property and make SVG path d attribute as the presentation attribute](https://bugzilla.mozilla.org/show_bug.cgi?id=1744599) this was an experimental feature introduced in FF91 in https://bugzilla.mozilla.org/show_bug.cgi?id=1340422

Looking at this the BCD is a bit messy. We need to update the d property and the `path()`. There is missing spec info and it is not clear whether it is still "partial".
Other than that it looks like we need to remove experimental feature and add the release note.

- [ ] BCD
- [ ] Docs
- [ ] Experimental features
- [ ] Release notes
- [ ] dev-doc-complete

**Comment by Rumyra** [comment]
Thanks @hamishwillee - I'm keeping my eye on the BCD for this (even tho it's making my head hurt) :)

**Comment by hamishwillee** [comment]
Cool. I'm chasing that up. This is pretty much done/on the path to done though.

**Comment by Rumyra** [comment]
In anticipation I'll close this :)
